### PR TITLE
Settings fix donate button

### PR DESCRIPTION
**Pull request by jsbd29**
The User Settings button on the dashboard will take the user to their settings. 
If the user changes their email or password, they will be redirected to the dashboard. 
Similarly, if the user confirms the deletion of their account, they will be redirected to the landing page.
Donate button view is created at DonateModal

### CODE REVIEW

File: ClientApp/src/.AppRoutes.js.swp
Comment (Shan0x) [inline ClientApp/src/.AppRoutes.js.swp:1]: Please remove.

File: ClientApp/src/AppRoutes_BACKUP_47497.js
Comment (Shan0x) [inline ClientApp/src/AppRoutes_BACKUP_47497.js:1]: Please remove.

File: ClientApp/src/AppRoutes_BASE_47497.js
Comment (Shan0x) [inline ClientApp/src/AppRoutes_BASE_47497.js:1]: Please remove.

File: ClientApp/src/AppRoutes_LOCAL_47497.js
Comment (Shan0x) [inline ClientApp/src/AppRoutes_LOCAL_47497.js:1]: Please remove.

File: ClientApp/src/AppRoutes_REMOTE_47497.js
Comment (Shan0x) [inline ClientApp/src/AppRoutes_REMOTE_47497.js:1]: Please remove.

**Review by Shan0x — CHANGES_REQUESTED** [review]

File: ClientApp/src/AppRoutes_BASE_47497.js
Comment (jsbd29) [reply ClientApp/src/AppRoutes_BASE_47497.js:1]: done

File: ClientApp/src/AppRoutes_LOCAL_47497.js
Comment (jsbd29) [reply ClientApp/src/AppRoutes_LOCAL_47497.js:1]: done

File: ClientApp/src/AppRoutes_REMOTE_47497.js
Comment (jsbd29) [reply ClientApp/src/AppRoutes_REMOTE_47497.js:1]: done

**Review by Shan0x — APPROVED** [review]
Looks good!

**Review by crjpb3 — APPROVED** [review]

**Pull request merged**